spec/amp-actions-and-events.md

```diff
@@ -324,7 +322,8 @@ Including: `input[type=radio]`, `input[type=checkbox]`, `input[type=range]`, and
   </tr>
 </table>
 
-## `AMP` target
+## Special targets
+### `AMP`
 
 `AMP` target is a special target. It's provided by the AMP runtime and implements top-level
```

Comment:
As you now have "Special" in the header, I suggest you rewrite this as:

The `AMP` target is provided by the AMP Runtime and ...